### ISSUE TITLE
Undefined index: ckeditor 

### ISSUE THREAD

**Issue by bjsmasth** [helios-ag/FMElfinderBundle#121]
I have used FMElfinderBundle and TrsteelCkeditorBundle and configure as mentioned in documentation. Unfortunately i have got Undefined index: ckeditor

**Comment by helios-ag** [comment]
Can you post your full configuration?

**Comment by Hakadel** [comment]
In config.yml

    ivory_ck_editor:
    default_config: default
    configs:
        default:
            filebrowserBrowseRoute: elfinder
            filebrowserBrowseRouteParameters: []

**Comment by GoZOo** [comment]
i have same error and solve it doing :
```
filebrowser_image_browse_url:
            route: elfinder
            route_parameters: []
```

**Comment by helios-ag** [comment]
Fixed readme in 6a04bbd6e5bbaaaa2d04848cd324b1cc6c4253db
